src/Console/Command/AddPrefixCommand.php

```
@@ -234,4 +251,76 @@ private function validateOutputDir(InputInterface $input, OutputStyle $io)
             $this->fileSystem->remove($outputDir);
         }
     }
+
+    private function validateReplaceStrings(InputInterface $input)
+    {
+        $replaceStrings = $input->getOption(self::REPLACE_STRINGS_OPT);
+
+        if (false === $this->fileSystem->isAbsolutePath($replaceStrings)) {
+            $replaceStrings = getcwd().DIRECTORY_SEPARATOR.$replaceStrings;
+        }
+
+        $input->setOption(self::REPLACE_STRINGS_OPT, $replaceStrings);
+
+        if (false === $this->fileSystem->exists($replaceStrings)) {
+            return;
```

Should raise an exception actually...